src/pet.js

```diff
@@ -0,0 +1,125 @@
+// CONSTRUCTOR FOR PET OBJECTS //
+const PET_IS_BORN = 0;
+const MAXIMUM_HUNGER = 10;
+const MINIMUM_FITNESS = 0;
+
+function Pet(name) {
+    this.name = name;
+    this.age = PET_IS_BORN;
+    this.hunger = MINIMUM_HUNGER;
+    this.fitness = MAXIMUM_FITNESS;
+    this.children = [];
+};
+//GETTERS//
+//ISALIVE: Checks states to determine if pet is alive//
+
+const LIFESPAN = 30;
```

Comment:
Its a good practice to make sure all your constants are in the same place, this one and all the ones below should be moved to the top of the file with all the others

Regards the naming of he constants, its not clear at first glance what they represent. What do PET_IS_BORN / A_YEAR / SOFA_TIME / FASTING / A_WORKOUT / A_SNACK actually mean? You arent limited to a character length so you can afford to be a bit more descriptive with these